cutter/cutter/main.py

```diff
@@ -332,6 +339,10 @@ def set_row(**kwargs):
 			))
 			result = query(self.conn, built_query, id=job.id, name=self.name, **kwargs)
 			if result.rowcount != 1:
+				# If we hadn't yet set finalizing, then this means an operator cancelled the job
```

Comment:
To be pedantic, the lack of a row _likely_ means the job has been cancelled. The is still a very slim chance that a bug or database error could cause the row to go missing.